.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
-  "eslint.enable": true,
-  "deepscan.enable": true
-}
\ No newline at end of file
+  "deepscan.enable": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Is this needed?

Reply:
Well I just saw expectations of prettier to work in a test file and the presence of a .vscode folder so I added it - it is not required no